### ISSUE TITLE
KBase SDK Tests action fails

### ISSUE THREAD

**Issue by nilsoberg** [kbase-sfa-2021/example_sdk_app#18]
The "KBase SDK Tests" action always fails.  On the initial commit, the output is 

```
Run sh $GITHUB_WORKSPACE/kb_sdk_actions/bin/kb-sdk test

Validating module in (/home/runner/work/efi_family_app/efi_family_app)

Validating method in (/home/runner/work/efi_family_app/efi_family_app/ui/narrative/methods/run_example_kb_sdk_app)


Congrats- this module is valid.

Error while testing module: Error: KBase account credentials are not set in test_local/test.cfg

For more help and usage information, run:
    kb-sdk help

Error: Process completed with exit code 1.
```

On successive commits the test also fails.  The output is attached.  The most specific error is `User nilsoberg may not read workspace`.

[kbase_sdk_test.txt](https://github.com/kbase-sfa-2021/example_sdk_app/files/11202053/kbase_sdk_test.txt)

**Comment by bio-boris** [comment]
This is expected. You need to add your kbase developer token to github
<img width="786" alt="image" src="https://user-images.githubusercontent.com/1258634/231266498-6d653981-0dab-4481-9042-242662dc38e5.png">

**Comment by bio-boris** [comment]
See https://github.com/kbase-sfa-2021/example_sdk_app/blob/main/.github/README.md

**Comment by bio-boris** [comment]
`User nilsoberg may not read workspace.`

* Workaround is to use an object that is not in a public workspace
* Fix  for us is to change the tests to test an object in a public workspace

**Comment by nilsoberg** [comment]
> This is expected. You need to add your kbase developer token to github <img alt="image" width="786" src="https://user-images.githubusercontent.com/1258634/231266498-6d653981-0dab-4481-9042-242662dc38e5.png">

I agree about the secret.  However, the error I quoted above is generated when the template is initially used, and there's no opportunity to add the secret.  Additionally, each test is run twice in the actions list.  Maybe I'm misunderstanding things?

**Comment by bio-boris** [comment]
* Its not possible to add the secret before the repository exists. It has to exist to add the secret. I guess you could start with a blank repo and populate the secret, but its much easier to use as a template and then add the secret. 
* The test runs under the following conditions and can be customized when running

https://github.com/kbase-sfa-2021/example_sdk_app/blob/main/.github/workflows/kb_sdk_test.yaml#L3

**Comment by nilsoberg** [comment]
Thanks for the info. I want to use the template, all I'm saying is that the test fails when the template is initially "used".  The error is `
Error while testing module: Error: KBase account credentials are not set in test_local/test.cfg`.  Once the token is set, the Action tests succeed.  Just not on the very first one when GitHub is initially setting up the repo from the template.
